ffmeta/data/ffmetadata_ddl.sql

```diff
@@ -60,122 +60,131 @@ CREATE TABLE IF NOT EXISTS `raw2` (
   `wave` char(30) CHARACTER SET latin1 DEFAULT NULL,
   `section` char(1) DEFAULT NULL,
   `leaf` char(30) DEFAULT NULL,
+  `obs` int(4) DEFAULT NULL,
+  `min` int(7) DEFAULT NULL,
+  `max` int(20) DEFAULT NULL,
+  `avg` float(7) DEFAULT NULL,
```

Comment:
Unless you have good reason to specify the precision of the floating-point fields, I'd just keep this as a simple `float` (or `double`). If you do need a precision value, then probably better to use the same value for all `float` fields you've added.